try {
 console.log("hello");
 console.log(a);
} catch {
  console.log("Handled");
} finally {

}